### PR TITLE
MM-18881 - Attempting to configure uninstalled plugins gets stuck at Loading... without timeout.

### PR DESCRIPTION
**Pull request by ali-farooq0**
#### Summary
Emitting `notifyPluginStatusesChanged` when a plugin is removed.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-18881

### CODE REVIEW

**Comment by marianunez** [comment]
Should we include a test case for this ?

**Comment by ali-farooq0** [comment]
> Should we include a test case for this ?

Unfortunately the way everything is set up with websocket events, its sometimes 
 hard to test these type of changes. However, I've still added a test, lets see if it causes any flakiness :)

File: api4/plugin_test.go [hunk 1]
```diff
@@ -305,11 +305,33 @@ func TestNotifyClusterPluginEvent(t *testing.T) {
 	require.Equal(t, "testplugin", manifest.Id)
 
 	// Successful remove
+	webSocketClient, err := th.CreateWebSocketSystemAdminClient()
+	require.Nil(t, err)
+	webSocketClient.Listen()
```
Comment (marianunez) [inline api4/plugin_test.go]: Should we include a defer for `websocketClient.Close()` ?

Comment (ali-farooq0) [reply api4/plugin_test.go]: Great catch! Updated.

**Pull request merged**